src/Xamarin.Android.Tools.Bytecode/XmlClassDeclarationBuilder.cs

```diff
@@ -427,6 +429,16 @@ IEnumerable<XElement> GetExceptions (MethodInfo method)
 			}
 		}
 
+		static XAttribute? GetAnnotatedVisibility (AttributeCollection attributes)
+		{
+			var annotations = attributes?.OfType<RuntimeInvisibleAnnotationsAttribute> ().FirstOrDefault ()?.Annotations;
+
+			if (annotations?.FirstOrDefault (a => a.Type == "Landroidx/annotation/RestrictTo;") is Annotation annotation)
+				return new XAttribute ("annotated-visibility", ((annotation.Values.FirstOrDefault ().Value as AnnotationElementArray)?.Values?.FirstOrDefault () as AnnotationElementEnum)?.ConstantName ?? string.Empty);
```

Comment:
[`RestrictTo.getValue()`](https://developer.android.com/reference/androidx/annotation/RestrictTo#getValue()) returns an *array* of `RestrictTo.Scope` values; why pick just the first value instead of printing a ` `-separated sequence of all of them?

Reply:
I don't think I realized it was an array.  By the time I figured out that monstrosity of code needed to get a value I thought I had the only value.  😁